src/include/columnar/cstore.h

```diff
@@ -34,6 +34,8 @@
 #define STRIPE_ROW_COUNT_MAXIMUM 10000000
 #define CHUNK_ROW_COUNT_MINIMUM 1000
 #define CHUNK_ROW_COUNT_MAXIMUM 100000
+#define COMPRESSION_LEVEL_MIN 1
+#define COMPRESSION_LEVEL_MAX 19
```

Comment:
Is this the same for lz4, lz4hc, and zstd?

If not it seems a little awkward to have a global max.

Reply:
Currently we only use compression level for zstd. I'll address your comment in a separate PR after looking if we can use compression level for other methods too.

Reply:
OK. I think we should make the min/max the largest min/max of all compression methods supported, and then just clamp it down to the max of the specific compression method at the time the setting is stored for the table. In other words, if lz4 only supports 1..9, and someone sets it to 19, it wouldn't error out; it would adjust it to 9 before storing in the table's settings (though we should issue a WARNING).

The reason is that I'd like to avoid introducing errors or complex validation checking. Thoughts?